docs/hassio_addon_security.md

```diff
@@ -15,3 +28,4 @@ As a developer, follow the following best practices to make your add-on secure:
 - Don't run on host network
 - Create an AppArmor profile
 - Map folders read only if you don't need write access
+- If you need any API access, make sure you that you not grant to highest permission if you don't need it
```

Comment:
If you need to access an API, make sure that you not grant a higher permission than what you need.